order/src/main/java/org/order/OrderService.java

```diff
@@ -0,0 +1,34 @@
+package org.order;
+
+import jakarta.transaction.Transactional;
+import java.util.List;
+import java.util.stream.Collectors;
+import lombok.AllArgsConstructor;
+import org.order.dto.PurchaseOrderDto;
+import org.order.entity.PurchaseOrder;
+import org.springframework.stereotype.Service;
+
+@Service
+@AllArgsConstructor
```

Comment:
Prefer to use @RequiredArgsConstructor if you want to create a constructor with your dependencies. RequiredArgs will only create constructor for the field with final or @NotNull on it, thanks to this we can , somehow, make sure that ours beans are final which is a good thing
```suggestion
@RequiredArgsConstructor
```